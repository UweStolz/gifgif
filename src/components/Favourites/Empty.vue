<template functional>
  <v-container
    fluid
    fill-height
  >
    <v-col
      class="text-center "
      cols="12"
    >
      <v-sheet
        class="display-2"
        tile
        width="100%"
        height="100%"
      >
        Nothing to show here ☹️
      </v-sheet>
    </v-col>
  </v-container>
</template>

<script lang="ts">
import Vue from 'vue';

export default Vue.extend({
  name: 'Empty',
  functional: true,
});
</script>
