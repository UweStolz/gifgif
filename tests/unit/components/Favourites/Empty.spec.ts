import Empty from '@/components/Favourites/Empty.vue';
import shallow from '../../../helper';

describe('Empty.vue', () => {
  it('renders properly', () => {
    const wrapper = shallow(Empty);
    expect(wrapper.element).toMatchSnapshot();
  });
});
