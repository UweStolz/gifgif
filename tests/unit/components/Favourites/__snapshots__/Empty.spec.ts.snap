// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty.vue renders properly 1`] = `
<div
  class="container container--fluid fill-height"
>
  <div
    class="text-center col col-12"
  >
    <div
      class="display-2 v-sheet theme--light rounded-0"
      style="height: 100%; width: 100%;"
    >
      
      You have no favourites yet ☹️
    
    </div>
  </div>
</div>
`;
